stepper-lib/src/air/fold/iterable.rs

```diff
@@ -0,0 +1,250 @@
+/*
```

Comment:
Love this code! Looks and reads really nice :)

Reply:
Hope, that after decopling boxed types instead of raw JValue's there will be more abstractions on it like this.